### ISSUE TITLE
How did you figure out the bedrock connection monitoring setup?

### ISSUE THREAD

**Issue by joebywan** [doctorray117/minecraft-ondemand#46]
I've currently used your setup for a minecraft server which has been great, but I'm trying to wander off into the weeds by setting up a valheim server.

It uses 2456-2457/udp.

I was looking at using something like a python library to query the server direct, but then I realised your bedrock code is detecting connections on UDP.

Spent the evening trying to understand it, currently lost on the bedrockping string you build.

Basically just seeing if you'd be willing to shed some light on it before I spend some more time on it :)

Thanks again.

**Comment by joebywan** [comment]
Ahh cool. So you're emulating a client trying to query the server, then parsing the response to get the current player count yeah?

Thanks for that link to gamedig, I'll check it out.

**Comment by joebywan** [comment]
I've been playing with trying to build a container around aws-cli and adding nvm, node & npm to get gamedig on it.  Have you managed to get it working?
